### PR TITLE
Remove use of vaParent for v2 facility page and throughout flow where unneeded

### PR DESCRIPTION
**Pull request by narin**
## Description
`vaParent` is really only needed for v1 of the facility selection page and is causing issues on the v2 page for unique cases like Sacramento.  This PR removes or replaces the use of `vaParent` throughout the flow, mainly by replacing the selectors that rely on it, such as `getRootIdForChosenFacility()` and `getSiteIdFromOrganization()` and `getRootIdOfOrganization()`.  From my understanding, the first 3 digits of a facility id should always produce the siteId/rootId.  I'm unsure if this has any implications for FHIR since it is no longer extracting the ids from the identifiers, so this may need to be revisited once we resume FHIR/Cerner integration.

## Testing done
Local and unit

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/utils/selectors.js
Comment (jbalboni) [inline src/applications/vaos/utils/selectors.js:191]: Is this right? Aren't all of the FHIR ids prefixed with `var`?

File: src/applications/vaos/services/location/index.js
Comment (jbalboni) [inline src/applications/vaos/services/location/index.js:220]: Related to your question about this maybe not being in line with FHIR, because we're parsing these ids, I think we could replace some uses of this function with a function that pulls the id out of the `identifiers` list in a Location resource, then do the slice down to 3 digits, and that would I think be enough to keep ourselves in good shape related to FHIR data.

I think we can leave that to a separate PR, though.

File: src/applications/vaos/utils/selectors.js
Comment (narin) [reply src/applications/vaos/utils/selectors.js:191]: Ah, you're right.  This selector is actually no longer used due to the changes in this PR.  Removing it entirely.

File: src/applications/vaos/utils/selectors.js
Comment (jbalboni) [inline src/applications/vaos/utils/selectors.js:74]: Is this flag added in other PRs? I don't think it's in the featureFlagNames.js file.

Comment (narin) [reply src/applications/vaos/utils/selectors.js:74]: Added to `featureFlagNames.js`.  Corresponding `vets-api` PR is here: https://github.com/department-of-veterans-affairs/vets-api/pull/5286.  Did I miss any others?  Haven't created a feature flag before.

Comment (jbalboni) [reply src/applications/vaos/utils/selectors.js:74]: Nope, those are the two pieces.

File: src/applications/vaos/utils/selectors.js
Comment (jbalboni) [inline src/applications/vaos/utils/selectors.js:78]: You can simplify this by doing `(!selectIsRegisteredToSacramentoVA(state) || vaosFlatFacilityPageSacramento(state))`. You don't need the extra `selectIsRegisteredToSacramentoVA` check, that has to be true to get to the logic after the `||`.

**Review by jbalboni — APPROVED** [review]
One other minor comment, and we should wait for the backend PR, but LGTM.

**Comment by narin** [comment]
@department-of-veterans-affairs/frontend-review-group Can I get a quick review on this whenever you get a chance? Thanks!

**Review by U-DON — APPROVED** [review]

**Pull request merged**
